### ISSUE TITLE
Use template syntax instead of @before and @after

### ISSUE THREAD

**Issue by hanneskod** [hanneskod/readme-tester#2]
``` php
/**
 * Headline (or {{ignore}} or similar to ignore headline)
 * (or use markdown directly, eg. # Headline)
 * 
 * Before example text
 * {{example}}
 * After example text
 *
 * Print the test return/output value
 * {{return}}
 */
public function exampleOne()
{
    // void example
} 
```

**Comment by hanneskod** [comment]
A similar syntax can then be used to build readme.md. In something like readme.src.md you could write.

```
# My cool project

Some explanations...

All examples from IntegrationExamples.php, including descriptions in class docbloc.
{{IntegrationExamples}} 

Just one example
{{SyntaxExamples::exampleOutput}}
```

**Comment by hanneskod** [comment]
`README.md` can be a tested in itself using html comments to hide testing instructions from readers:

    
    ```php
    // This test pass becouse the correct output is generated
    echo "foobar";
    ```

This syntax requires an exemplify comment directly followed by a php-colorized code block. The code block is executed in isolation and the defined expectation is validated.

Then some kind of phpunit hook is required to automatically test `README.md` as normal testing occurs (with the possibility to set what files are to be tested).

The executable would then work as an alternative way of testing examples.

**Comment by hanneskod** [comment]
Can be implemented using a test listener. See `PHPUnit_Framework_BaseTestListener`.

https://phpunit.de/manual/current/en/extending-phpunit.html

and

https://phpunit.de/manual/current/en/appendixes.configuration.html#appendixes.configuration.test-listeners

**Comment by hanneskod** [comment]
Closed in favour of #4.
